frontend/src/locales/scripts/en.json5

```
@@ -80,14 +80,14 @@
     },
     collection: {
       content: {
-        a: "{openverse} searches across more than 700 million images and audio from open APIs and the {commonCrawl} dataset.",
+        a: "{openverse} searches across more than 700 million images and audio track from open APIs and the {commonCrawl} dataset.",
         b: "We aggregate works from multiple public repositories, and facilitate reuse through features like one-click attribution.",
       },
     },
     planning: {
       content: {
         /** about.planning.content.a-c are parts of a single statement */
-        a: "Currently {openverse} only searches images and audio, with search for video provided through External Sources.",
+        a: "Currently {openverse} only searches images and audio track, with search for video provided through External Sources.",
```

Please use the plural form.

```suggestion
        a: "Currently {openverse} only searches images and audio tracks, with search for video provided through External Sources.",
```